GitUI/UserControls/ToolStripGitStatus.cs

```diff
@@ -103,10 +103,11 @@ public ToolStripGitStatus()
             _gitDirWatcher.NotifyFilter = NotifyFilters.FileName | NotifyFilters.LastWrite;            
         }
 
-        private void GitUICommandsChanged(IGitUICommandsSource source, GitUICommands oldCommands)
+        private void GitUICommandsChanged(object sender, GitUICommandsChangedEventArgs e)
         {
-            if (oldCommands != null)
+            if (e != null)
```

Comment:
:exclamation: `e` should never be null. Pass a non-null instance of the `EventArgs` class even if the property it contains is null.